src/group/membership/components/GroupMemberLeave.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import fp from 'lodash/fp';
+import { useTranslation } from 'react-i18next';
+
+import ConfirmButton from 'common/components/ConfirmButton';
+
+const GroupMemberLeave = props => {
+  const { t } = useTranslation();
+  const { label, owner, onConfirm, buttonProps, loading } = props;
+
+  return (
+    <ConfirmButton
+      onConfirm={onConfirm}
+      confirmTitle={t('group.membership_leave_confirm_title', {
+        name: fp.get('name', owner),
```

Comment:
@paulschreiber This is the strict functional programming methods lodash expose, I create a PR to move everything lo it https://github.com/techmatters/terraso-web-client/pull/175 this is important to avoid unintended mutations, so after this is merged all of them will be _